data-serving/scripts/converters.py

```diff
@@ -343,14 +348,89 @@ def convert_outbreak_specifics(id: str, reported_market_exposure: str,
         normalized = convert_bool(lives_in_wuhan)
         if normalized is not None:
             outbreak_specifics['livesInWuhan'] = normalized
-    except (ValueError):
+    except ValueError:
         logging.warning(
             '[%s] [outbreakSpecifics[livesInWuhan]] invalid value %s', id,
             lives_in_wuhan)
 
     return outbreak_specifics if outbreak_specifics else None
 
 
+def convert_location_string(location_str: str) -> Dict[str, Any]:
+    if pd.isna(location_str) or location_str.lower() == 'no' or location_str.lower() == 'none':
+        return None
+    if type(location_str) is not str:
+        raise ValueError('location is not a string')
+    if ';' in location_str or ':' in location_str or location_str.count(',') > 1:
+        raise ValueError('location is a list')
+
+    # Remove common prefixes 'travelled to' and 'traveled to' and lowercase it.
+    normalized_location = re.sub(
```

Comment:
Higher-level question: is this conversion approach (for location) the right strategy? This is a low precision PR, but it's already feeling a little belabored.

Are we ever going to do a good enough job coercing it to our validated schema? Should we just let old data lie?

Reply:
TBH I don't know... hoping to get more clarity on this from UXR/product. It seems like the line-list data may be used for research, in which case I do think it would be worthwhile to try to get as much coverage as possible... but yeah, basically need to understand where on the ROC curve we want to be, and how much time we want to invest in getting there :)

Reply:
FYI added a change to the README with these open q's so they're documented somewhere for now

Reply:
SG, let's revisit after we get more information from UXR/PM.